api/rds.proto

```diff
@@ -327,8 +327,14 @@ message RateLimit {
 message HeaderValue {
   // Header name.
   string key = 1;
-  // Header value.
+
+  // Header value. The same format specifier as used for HTTP access logging
+  // applies here [V2-API-DIFF].
   string value = 2;
+
+  // Should the value be appended? If false (default), the value overrides
+  // existing values [V2-API-DIFF].
+  google.protobuf.BoolValue append = 3;
```

Comment:
if false is default I think this can just be a normal primitive? Or are we doing wrappers for all booleans?